build-scripts/requirements.sh

```diff
@@ -38,6 +38,17 @@ fi
 set -x
 if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
     $travis_retry sudo apt-get --yes install parallel
+
+    if [[ "$LIBBACKTRACE" == "1" ]]; then
+        git clone https://github.com/ianlancetaylor/libbacktrace.git
+        (
+            cd libbacktrace
+            git checkout 4d2dd0b172f2c9192f83ba93425f868f2a13c553
```

Comment:
Why do a `checkout` here if this commit is the latest on ianlancetaylor's repo
https://github.com/ianlancetaylor/libbacktrace/commit/4d2dd0b172f2c9192f83ba93425f868f2a13c553